tests/console/sshd.pm

```diff
@@ -97,45 +94,55 @@ sub run {
-    assert_script_run "( ssh -NL 4242:localhost:22 $ssh_testman\@localhost & )";
-    assert_script_run "( ssh -NR 0.0.0.0:5252:localhost:22 $ssh_testman\@localhost & )";
+    assert_script_run "echo 'sshd.pm: Testing port forwarding' | logger";
+    assert_script_run "( ssh -vNL 4242:localhost:22 $ssh_testman\@localhost & )";
+    assert_script_run "( ssh -vNR 0.0.0.0:5252:localhost:22 $ssh_testman\@localhost & )";
     assert_script_run 'until ss -tulpn|grep sshd|egrep "4242|5252";do sleep 1;done';
+
+    # Scan public keys on forwarded ports
     assert_script_run "ssh-keyscan -p 4242 localhost >> ~/.ssh/known_hosts";
     assert_script_run "ssh-keyscan -p 5252 localhost >> ~/.ssh/known_hosts";
 
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost whoami";
-    assert_script_run "ssh -p 5252 $ssh_testman\@localhost whoami";
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 22 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 4242 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 5252 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
+    # Connect to forwarded ports
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost whoami";
+    assert_script_run "ssh -v -p 5252 $ssh_testman\@localhost whoami";
 
-    assert_script_run "ssh -p 4242 -tt $ssh_testman\@localhost ssh -tt $ssh_testman\@localhost whoami";
-    assert_script_run "ssh -t -o ProxyCommand='ssh $ssh_testman\@localhost nc localhost 4242' $ssh_testman\@localhost whoami";
+    # Copy the list of known hosts to $ssh_testman's .ssh directory
+    assert_script_run "install -m 0400 -o $ssh_testman ~/.ssh/known_hosts /home/$ssh_testman/.ssh/known_hosts";
+
+    # Test SSH command within SSH command
+    assert_script_run "ssh -v -p 4242 -tt $ssh_testman\@localhost ssh -tt $ssh_testman\@localhost whoami";
+
+    # Test ProxyCommand option
+    assert_script_run "ssh -v -t -o ProxyCommand='ssh -v $ssh_testman\@localhost nc localhost 4242' $ssh_testman\@localhost whoami";
+
+    # Test JumpHost option
     if (is_leap('15.0+') || is_tumbleweed || is_sle('15+')) {
-        assert_script_run("ssh -J $ssh_testman\@localhost:4242 $ssh_testman\@localhost whoami");
+        assert_script_run("ssh -v -J $ssh_testman\@localhost:4242 $ssh_testman\@localhost whoami");
     }
 
     # SCP (poo#46937)
+    assert_script_run "echo 'sshd.pm: Testing SCP subsystem' | logger";
     assert_script_run "scp -4v $ssh_testman\@localhost:/etc/resolv.conf /tmp";
     assert_script_run "scp -4v '$ssh_testman\@localhost:/etc/{group,passwd}' /tmp";
     assert_script_run "scp -4v '$ssh_testman\@localhost:/etc/ssh/*.pub' /tmp";
 
     # poo#80716 Test all available ciphers, key exchange algorithms, host key algorithms and mac algorithms.
+    assert_script_run "echo 'sshd.pm: Testing cryptographic policies' | logger";
```

Comment:
How will you use this logger file afterwards?

Reply:
ah they are visible through journalctl, all good

Reply:
From my experience with bsc#1175310 bsc#1181308 it is quite hard to read the journal log and find corresponding line in the test (as there was segfault but no red box as the sshd tries to recover from such issues) so this output is like anchor. I tried to no overuse it so I put only few of them.